### PR TITLE
feat: Add `shutdown_script_timeout`

### PR DESCRIPTION
**Pull request by mafredri**
Ref: https://github.com/coder/coder/pull/6139

### CODE REVIEW

File: docs/data-sources/parameter.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Optional:
 - `error` (String) An error message to display if the value doesn't match the provided regex.
 - `max` (Number) The maximum of a number parameter.
 - `min` (Number) The minimum of a number parameter.
+- `monotonic` (String) Number monotonicity, either increasing or decreasing.
```
Comment (mtojek) [inline docs/data-sources/parameter.md:59]: 👍 

nit: we may want to improve the Makefile to detect discrepancies.

**Review by mtojek — APPROVED** [review]
It's complimentary to the startup timeout, so 👍 .

File: docs/data-sources/parameter.md (hunk 1 above)
Comment (mafredri) [reply docs/data-sources/parameter.md:59]: This is done via `make gen`, we should probably verify in CI that there are no changes.

Comment (mtojek) [reply docs/data-sources/parameter.md:59]: Yes, that's what I meant.

Comment (mafredri) [reply docs/data-sources/parameter.md:59]: I'll toss up a PR 👍🏻

**Pull request merged**